backend-base/src/main/resources/application.properties

```diff
@@ -1,4 +1,6 @@
 # Configuration file
 # key = value
 quarkus.swagger-ui.always-include=true
-quarkus.ssl.native=true
\ No newline at end of file
+quarkus.ssl.native=true
+che.api=http://che-che.apps-crc.testing/api
```

Comment:
These 2 lines certainly make tests easier, but might bring failure at runtime. They should not be made available to the real application.